tests/instancetype_test.go

```diff
@@ -192,6 +192,68 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			Expect(cause.Field).To(Equal("spec.preference"))
 		})
 	})
+	Context("[Serial]with cluster memory overcommit being applied", Serial, func() {
+		BeforeEach(func() {
+			kv := util.GetCurrentKv(virtClient)
+
+			config := kv.Spec.Configuration
+			config.DeveloperConfiguration.MemoryOvercommit = 200
+			tests.UpdateKubeVirtConfigValueAndWait(config)
+		})
+		AfterEach(func() {
+			By("Cleaning Cluster MemoryOvercommit")
+			kv := util.GetCurrentKv(virtClient)
+
+			config := kv.Spec.Configuration
+			config.DeveloperConfiguration.MemoryOvercommit = 100
+			tests.UpdateKubeVirtConfigValueAndWait(config)
```

Comment:
kubevirt spec is always restored during global `AfterEach` function for serial tests, hence this can be dropped.